package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">  
+  <name>small_gicp</name>
+  <version>0.1.0</version>
+  <description>Efficient and parallelized algorithms for point cloud registration</description>
+
+  <maintainer email="k.koide@aist.go.jp">Kenji Koide</maintainer>
+  <license>MIT</license>
+
+  <buildtool_depend>cmake</buildtool_depend>
+  <depend>eigen</depend>
+  <depend>libomp-dev</depend>
+  <depend>libpcl-all-dev</depend>
+  <depend>nanoflann</depend>
+  <depend>sophus</depend>
```

Comment:
```suggestion
```
Both nanoflann and Sophus are vendored with the library and significantly extended/customized.